packages/react-kit/src/components/modal/components/Commit/CommitNonModal.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import { useAccount } from "../../../../hooks/connection/connection";
+import { useDisconnect } from "../../../../hooks/connection/useDisconnect";
+import { ReturnUseProductByUuid } from "../../../../hooks/products/useProductByUuid";
+import { theme } from "../../../../theme";
+import { Exchange } from "../../../../types/exchange";
+import { VariantV1 } from "../../../../types/variants";
+import Loading from "../../../ui/loading/Loading";
+import NonModal, { NonModalProps } from "../../nonModal/NonModal";
+import { BosonFooter } from "../common/BosonFooter";
+import { PurchaseOverviewView } from "../common/StepsOverview/PurchaseOverviewView";
+import { CommitSuccess } from "./CommitSuccess/CommitSuccess";
+import { ContractualAgreementView } from "./ContractualAgreementView/ContractualAgreementView";
+import { LicenseAgreementView } from "./LicenseAgreementView/LicenseAgreementView";
+import { OfferFullDescriptionView } from "./OfferFullDescriptionView/OfferFullDescriptionView";
+import { CommitOfferPolicyView } from "./OfferPolicyView/CommitOfferPolicyView";
+import { OfferVariantView, OfferVariantViewProps } from "./OfferVariantView";
+import {
+  DetailContextProps,
+  DetailViewProvider
+} from "../common/detail/DetailViewProvider";
+
+const colors = theme.colors.light;
+enum ActiveStep {
```

Comment:
Nitpick/Suggestion: Perhaps it would be more applicable to name this `Step` or `CommitStep`. IMO it is a bit confusing to have a en enum called ActiveStep used for a variable `previousStep` .